content/en/docs/a-warm-welcome-to-asn1-and-der.md

```diff
@@ -179,7 +181,7 @@ Again, lots of time types: UTCTime, GeneralizedTime, DATE, TIME-OF-DAY,
 DATE-TIME and DURATION. For HTTPS certificates you only have to care
 about UTCTime and GeneralizedTime.
 
-UTCTime represents a date and time as YYMMDDhhmm\[ss\], with an optional
+UTCTime represents a date and time as YYMMDDhhmmss, with an optional
```

Comment:
BER does allow the seconds to be optional here.  Though if you want to avoid the complexities of BER's flexibility, I won't fault you for it :smile:.  Certainly no one really needs to know about GeneralizedTime 46.2(a)(2)'s yyyyMMddHHmm with fractional minutes.

Reply:
Yeah, that was basically my thought process: I didn't want to get into the full detail in this section of what's optional and not (and what's allowed by BER vs DER), and figured it was misleading to mark the seconds as optional and not the minutes. But I'll keep the text as is and just mark the seconds optional. Thanks for the input!